### PR TITLE
change title description

### PR DESCRIPTION
**Pull request by XimenaCor**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,10 +6,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./styles.css"></link>
-    <title>Document</title>
+    <title>PRACTICA DE GIT</title>
 </head>
 
 <body>
+    <h4>GIT</h4>
```
Comment (Taxxx) [inline index.html]: please update to h3 instead of h4

**Review by Taxxx — CHANGES_REQUESTED** [review]
please see my observations

File: index.html (hunk 1 above)
Comment (XimenaCor) [reply index.html]: Done

**Review by Taxxx — APPROVED** [review]
LGTM!

**Pull request merged**